package.json

```diff
@@ -46,6 +46,8 @@
     },
     "scripts": {
         "test": "grunt test",
-        "prepare": "grunt build"
+        "prepare": "grunt build",
+        "build": "grunt build",
+        "dev": "grunt connect watch"
```

Comment:
I am actually not sure. I am using this to have a live developer session, so I used this. If its better to use `dev`, why not.

Reply:
`grunt dev` is the same as `grunt build connect watch`. There's no need to have a `dev` here in the package scripts (or a `build`), but if you want them there as an alias, `dev` should map to `grunt dev` to avoid confusion.